src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/IndexConfiguration/AbstractPimCatalogIntegration.php

```diff
@@ -103,14 +103,14 @@ protected function getSearchQueryResults(array $query)
     /**
      * Checks that the products found are effectively expected
      *
-     * @param array $productsFound
-     * @param array $expectedProducts
+     * @param array $productIdsFound
+     * @param array $expectedProductIds
      */
-    protected function assertProducts(array $productsFound, array $expectedProducts)
+    protected function assertProducts(array $productIdsFound, array $expectedProductIds)
```

Comment:
if you want to respect PHPunit semantic, it would be `$actualProductIds`

and maybe _identifiers_ is better than _ids_ :)